Docker/start-testnet.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -e
+
+function call_hook {
+    [[ -x hooks.sh ]] && ./hooks.sh "$@"
```

Comment:
Where can I find hooks.sh?

Reply:
The idea of the `hooks.sh` script is to supplement the startup sequence depending on the testnet environment. So, for a public testnet, after the launch of contracts, it is necessary to provide the `readuser` with access to the database of contracts. For a private testnet, this is not required.